samples/newChat/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>QuickBlox chat code sample</title>
+    <link rel="shortcut icon" href="https://quickblox.com/favicon.ico">
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <script src="./js/libs/path.min.js" defer></script>
```

Comment:
Recommend use a cdn for other libs.
It's also made our sample simplier)
https://cdnjs.com/libraries/underscore.js/
https://cdnjs.com/libraries/path.js/
https://cdnjs.com/libraries/meyer-reset